### DISCUSSION TITLE
Restart all services?

### DISCUSSION THREAD

**Discussion by rickalan** [Security-Onion-Solutions/securityonion#3555] [Unsupported Versions]
Is there a command to restart all the services on SO 2.3 without rebooting the whole box?

**Comment by weslambert** [comment]
Not at the moment, but you could do something quick like the following (not optimized):

`for i in $(docker ps | awk '{print $1}' | grep -v CONTAINER); do docker stop $i && docker rm $i; done && salt-call state.apply highstate`

We are tracking this via the following issue:

https://github.com/Security-Onion-Solutions/securityonion/issues/383
